### PR TITLE
bug: fix randomness_to_field

### PR DESCRIPTION
**Pull request by feltroidprime**
In case of Cairo this will cut the 5 most significant bits (From 256 to 251) correctly.

Basically it is returning `hash & (2^251-1)` as expected now.

### CODE REVIEW

**Comment by MauroToscano** [comment]
Thanks Feltroid! Tests are failing, as expected. I'm reworking them, and I will submit a PR to your branch to fix them.

**Comment by MauroToscano** [comment]
I have created a PR to your branch to fix the tests, and fix the case for fields that has more than one empty byte here: 

https://github.com/feltroidprime/lambdaworks_cairo_prover/pull/1

This should fix this PR and then it can be reviewed and merged

File: src/lib.rs [hunk 1]
```diff
@@ -211,55 +211,86 @@ mod tests {
...
+            FF00FF00FF00FF00\
+            FF00FF00FF00FF00\
+            FF00FF00FF00FF00",
+        );
+
         let field_element: FE = randomness_to_field(&mut randomness);
-        let expected_fe = FE::from(2u64.pow(63));
-        let expected_fe = expected_fe * FE::from(2u64);
+
+        println!("{}", field_element.representative().to_string());
```
Comment (MauroToscano) [inline src/lib.rs:252]:
```suggestion
```

**Review by MauroToscano — APPROVED** [review]

**Review by schouhy — APPROVED** [review]
LGTM

**Review by Oppen — APPROVED** [review]

**Pull request merged**